client/components/phone-input/test/test-phone-number.js

```diff
@@ -308,7 +373,7 @@ describe( 'getUpdatedCursorPosition', () => {
 	} );
 
 	test( 'should return last position for deleting multiple characters in between special characters', () => {
-		equal( getUpdatedCursorPosition( '(802) 687-6234', '802-6234', 6 ), 4 );
+		equal( getUpdatedCursorPosition( '(802) 687-6234', '802-6234', 6 ), 3 );
```

Comment:
I think the expected position should be 3 here instead of 4 -- my reasoning is that the cursor should be at the right edge of a digit since that is where it would be after entering a single digit.